cmake/OpenCVFindLibsGrfmt.cmake

```diff
@@ -9,8 +9,8 @@ else()
   hunter_add_package(ZLIB)
   find_package(ZLIB "${MIN_VER_ZLIB}")
   if(ZLIB_FOUND AND ANDROID)
-    if(ZLIB_LIBRARIES STREQUAL "${ANDROID_SYSROOT}/usr/lib/libz.so" OR
-        ZLIB_LIBRARIES STREQUAL "${ANDROID_SYSROOT}/usr/lib64/libz.so")
+    if(ZLIB_LIBRARIES STREQUAL "${CMAKE_SYSROOT}/usr/lib/libz.so" OR
```

Comment:
Actually here we should always use ZLIB from Hunter. Should be protected with `NOT HUNTER_ENABLED`.